lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -213,6 +213,9 @@ def build_authorization_request(money, payment_method, options)
               add_hcg_additional_data(xml, options) if options[:hcg_additional_data]
               add_instalments_data(xml, options) if options[:instalments]
               add_moto_flag(xml, options) if options.dig(:metadata, :manual_entry)
+			  if three_d_secure = options[:three_d_secure]
```

Comment:
Hello Chandan!

The indentation is going to cause CI to fail because of the rubocop rules it enforces. Would you mind attempting to indent properly? TY!

Reply:
Hi Bertrand - This has been updated, Thanks for highlighting the same.